concepts/tuples/.meta/config.json

```diff
@@ -1,7 +1,8 @@
 {
     "blurb": "In Cairo, tuples are immutable, fixed-length compound data types that allow you to group values of varying types into a single structured entity, facilitating efficient and type-safe data management.",
     "authors": [
-        "rajeebkm"
+        "rajeebkm",
+        "0xNeshi"
```

Comment:
```suggestion
```
You're the sole author :) If nothing else, you can add me as a contributor for reviewing the PR